data/mods/gen3/random-teams.ts

```diff
@@ -395,7 +395,7 @@ export class RandomGen3Teams extends RandomGen4Teams {
 		// First, the high-priority items
 		if (species.name === 'Ditto') {
 			item = this.sample(['Metal Powder', 'Quick Claw']);
-		} else if (species.name === 'Farfetch\'d') {
+		} else if (toID(species.name) === 'farfetchd') {
```

Comment:
species.id here as well, but the rest of the code seems to be checking names and it would look out of place to do IDs, so think actually might be a better place to use the \u

Reply:
Swapped to \u2019 for the random-teams cases